docs/en/03_Upgrading/index.md

```diff
@@ -464,22 +458,19 @@ If you want your codebase to comply with the PSR-4 standard, make sure sub-direc
 The `add-namespace` command of the [upgrader tool](https://github.com/silverstripe/silverstripe-upgrader/) provides a feature
 to namespace your codebase and to automatic update references to those classes.
 
-```
-composer global require silverstripe/upgrader
+```bash
 upgrade-code add-namespace "App\\Web" ./mysite/code --recursive --write
 ```
 
 This task will do the following:
 * Add the given namespace to all files in the code class, and subdirectories
 * All references to classes in any namespaced files will be safely retained with additional `use` directives added as necessary
-* A `mysite/.upgrade.yml` file be created/updated to record the new fully qualified name of each class.
-
-`.upgrade.yml` will be used in later steps to update references to the old non-namespaced classes.
-
-If you want to do a dry-run, omit the `--write` option to see a preview of all changed project files.
+* A `mysite/.upgrade.yml` file be created/updated to record the new fully qualified name of each class. 
```

Comment:
missing a `will` in "file will be created/updated"

Reply:
Fixed